trunner/device.py

```diff
@@ -44,6 +64,110 @@ def run(self, test):
             self.serial.close()
 
 
+class GPIO:
+    def __init__(self, pin):
+        self.pin = pin
+
+        RPi.GPIO.setmode(RPi.GPIO.BCM)
+        RPi.GPIO.setwarnings(False)
+        RPi.GPIO.setup(self.pin, RPi.GPIO.OUT)
+
+    def high(self):
+        RPi.GPIO.output(self.pin, RPi.GPIO.HIGH)
+
+    def low(self):
+        RPi.GPIO.output(self.pin, RPi.GPIO.LOW)
+
+
+class IMXRT106xRunner(DeviceRunner):
+    SDP = 'plo-ram-armv7m7-imxrt106x.sdp'
```

Comment:
please add docstring regarding how RT106x should be connected to be useable with this class.

Should we make some runner for testing board connected directly to the PC?

Reply:
I added docstrings.

On PC I don't know how to solve problem with restarting board. Should we print message "Now you have 10 seconds to restart board manually"?

Reply:
I don't know how it should be done, I'm just stating the problem (and trigger discussion whether we would like to do it now - maybe we should ask if/how it might help the people working with rt1064 right now?).

Possible solutions I can think of:
 - "interactive mode" ("Restart board into serial-downloader and press ENTER" && wait)
 - (in the future) configurable restart method (eg. by JLink, by `psh reboot`, etc.)?

Reply:
Yeap, I was thinking about "interactive mode". In the next PR I will add building for rt1064 and then I will be thinking about handling rt1064 connected directly to the PC - is hard to write tests when we can't run them locally;).

Reply:
Ok, still it would be good to gather feedback from other people working on rt1064 ;).